packages/cli/src/prompt.js

```diff
@@ -0,0 +1,23 @@
+import { stdin, stdout } from 'process';
+import readline from 'readline';
```

Comment:
We are catching a lucky break here. Most commands retain `ses` because it is necessary to establish a CapTP connection to the daemon, but this one does not. `readline` is not compatible with SES because it retains `domain`. This will work for now, but it might be worth finding an alternative.

Reply:
I'll leave this alone for now, but we may roll our own SES-compatible `readline` later, or hope that Node removes the `domain` dependency from their implementation.